apps/web/components/DigitalIcelandNewsCard/DigitalIcelandNewsCard.tsx

```diff
@@ -21,58 +21,91 @@ interface ItemProps {
   description?: string | null
   tags: string[]
   href: string
+  mini?: boolean
+  page: number
 }
 
 export const DigitalIcelandNewsCard = (item: ItemProps) => {
   const { format } = useDateUtils()
 
-  return (
-    <LinkV2 href={item.href} className={styles.itemContainer}>
-      <Box
-        display="flex"
-        flexDirection="column"
-        flexWrap="nowrap"
-        rowGap={[2, 2, 2, 5]}
-        justifyContent="spaceBetween"
-        height="full"
-      >
-        <Stack space={2}>
-          <BackgroundImage
-            positionX="left"
-            backgroundSize="cover"
-            image={{ url: item.imageSrc }}
-            ratio="396:210"
-            boxProps={{
-              alignItems: 'center',
-              width: 'full',
-              display: 'inlineFlex',
-              overflow: 'hidden',
-              borderRadius: 'large',
-            }}
-          />
+  const formattedDate = item.date && format(new Date(item.date), 'do MMMM yyyy')
 
-          <Stack space={1}>
-            {item.date && (
-              <Text variant="h5" color="purple400">
-                {format(new Date(item.date), 'do MMMM yyyy')}
-              </Text>
-            )}
-            <Text variant="h3">{item.title}</Text>
-            <Text variant="medium">
-              {shortenText(item.description ?? '', 80)}
-            </Text>
-          </Stack>
-        </Stack>
-        <Box>
-          <Inline space={1}>
-            {item.tags.map((tag) => (
-              <Tag key={tag} disabled>
-                {tag}
-              </Tag>
-            ))}
-          </Inline>
-        </Box>
+  if ((item.page === 1 && item.mini) || item.page > 1) {
```

Comment:
I suggest we move this check up to the level above and not have the component know what page it is on. Let's just send down the mini prop that should be enough right?